cmd/machineset/main.go

```diff
@@ -174,3 +181,33 @@ func main() {
 	// Start the Cmd
 	log.Fatal(mgr.Start(signals.SetupSignalHandler()))
 }
+
+func NewClientBuilder() cluster.ClientBuilder {
+	return &newClientBuilder{}
+}
+
+type newClientBuilder struct {
+	uncached []client.Object
+}
+
+func (n *newClientBuilder) WithUncached(objs ...client.Object) cluster.ClientBuilder {
+	n.uncached = append(n.uncached, objs...)
+	return n
+}
+
+func (n *newClientBuilder) Build(cache cache.Cache, config *rest.Config, options client.Options) (client.Client, error) {
+	// Create the Client for Write operations.
+	config.Burst = 250
+	log.Printf("Config: %s", config.Burst)
+	log.Printf("Client options: %s", options)
+	c, err := client.New(config, options)
+	if err != nil {
+		return nil, err
+	}
+
+	return client.NewDelegatingClient(client.NewDelegatingClientInput{
+		CacheReader:     cache,
+		Client:          c,
+		UncachedObjects: n.uncached,
+	})
+}
```

Comment:
I may be missing it, but where is this being used? I can't see it

Reply:
I thought I deleted this.  This was the first attempt, because I missed the obvious rest config args to manager.New().  BTW, this seems to have no effect on the initial discovery client, even though this code is called, it's after the discovery starts.